addons/udes_stock_picking_batch/views/stock_picking_batch.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+
+        <!-- Stock picking batch form -->
+        <record id="stock_picking_batch_form" model="ir.ui.view">
+            <field name="name">udes_stock.stock.picking.batch.form</field>
+            <field name="model">stock.picking.batch</field>
+            <field name="inherit_id"
+                   ref="stock_picking_batch.stock_picking_batch_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='state']" position="replace">
```

Comment:
Similar to this thread: https://github.com/unipartdigital/udes-open/pull/597/files#r847543615

Just replace the `statusbar_visible` attribute, same goes for the `action_cancel` button too (just change `states` in this one)